### PR TITLE
Fresh layout sidebar

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

A nice new sidebar, as seen on TV.
Two intentional todos for another PR:
- project switcher,
- pinned dashboards,
- toolbar modal,
- mobile mode.
Screen Shot 2021-11-01 at 18 16 13

### CODE REVIEW

**Review by clarkus — CHANGES_REQUESTED** [review]
This is awesome. The only thing I noticed that's missing is the groupings in the left navigation. Maybe it's just missing from the screenshot, but the intent is to group features into sections based on the type of feature. 

Screen Shot 2021-11-01 at 1 00 36 PM

https://www.figma.com/file/NHPA2hVWwaKp4eTGWdigs9/Product-Concepts?node-id=735%3A1662

The order of nav options is slightly different, too, but I'm wondering if that was intentional? In the designs I have the toolbar settings listed before the project settings.

**Comment by Twixes** [comment]
Oops, forgot about sections!
The slightly different order came from the old `MainNavigation` code, adjusted it.

**Comment by Twixes** [comment]
Sidebar now:
Screen Shot 2021-11-02 at 16 52 07

**Review by clarkus — APPROVED** [review]
LGTM. I left a comment in figma, but if we're on not feeing certain about the groupings, we could roll back to a simple separator or rule between the configuration options and the other features.

**Comment by Twixes** [comment]
I also added the sidebar to the `Lemonade` story in Storybook if you wanted an interactive dummy.

File: frontend/src/layout/lemonade/TopBar/SitePopover.tsx [hunk 1]
```diff
@@ -37,7 +38,7 @@ function AccountInfo(): JSX.Element {
         <div className="AccountInfo">
             <ProfilePicture name={user?.first_name} email={user?.email} size="xl" />
             <div className="AccountInfo__identification SitePopover__main-info">
-                <strong>{user?.first_name}</strong>
+                <b>{user?.first_name}</b>
```
Comment (mariusandra) [inline frontend/src/layout/lemonade/TopBar/SitePopover.tsx:41]: [Why not](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#usage_notes) `strong`?

**Review by mariusandra — APPROVED** [review]
It all looks good to me, just a tiny question inline.

File: frontend/src/layout/lemonade/TopBar/SitePopover.tsx (hunk 1 above)
Comment (Twixes) [reply frontend/src/layout/lemonade/TopBar/SitePopover.tsx]: 👀 Interesting, I had a vague feeling that there was a difference but I wasn't sure. Went for `strong`.

**Comment by Twixes** [comment]
Moved to nameless sections as we discussed @clarkus
Zrzut ekranu 2021-11-3 o 11 30 45

**Pull request merged**